src/Skoruba.IdentityServer4.Admin/Scripts/App/components/Picker.es5.js

```diff
@@ -100,6 +109,26 @@ ko.components.register('picker', {
 				}
 		};
 
+		//Get suggested items - by default 5 items
+		this.getSuggestedItems = function () {
+
+			if (self.allowSuggestedItems() === false) return;
+
+			if (params.url) {
+				//start loading
+				self.loading(true);
+
+				//make ajax request and result add to search result
+				$.get(params.url + "=" + "&limit=" + self.topSuggestedItems(), function (data) {
```

Comment:
Use `URLSearchParams` for this, not string concatenation

Reply:
I have used instead of string string concatenation - object as second parameter in get method, Thx. :-)